ShellPkg/Library/UefiShellDebug1CommandsLib/EfiDecompress.c

```diff
@@ -85,10 +85,16 @@ ShellCommandRunEfiDecompress (
       ShellStatus = SHELL_INVALID_PARAMETER;
     } else {
       TempParam = ShellCommandLineGetRawValue (Package, 1);
-      ASSERT (TempParam != NULL);
+      if (TempParam == NULL) {
+        ASSERT (TempParam != NULL);
+        ShellPrintHiiEx (-1, -1, NULL, STRING_TOKEN (STR_GEN_PARAM_INV), gShellDebug1HiiHandle, L"efidecompress");
+        ShellStatus = SHELL_INVALID_PARAMETER;
+        return (ShellStatus);

```

Comment:
I think `Package` should still be freed. So a `goot Done` would be better I think

Reply:
Agreed, added.